session/pingpong/accountant_caller.go

```diff
@@ -202,6 +202,21 @@ func (ac *AccountantCaller) GetConsumerData(id string) (ConsumerData, error) {
 	return resp, nil
 }
 
+// GetProviderBeneficiary gets provider beneficiary address.
+func (ac *AccountantCaller) GetProviderBeneficiary(id string) (string, error) {
```

Comment:
It would be better to check this on blockchain, as the accountant endpoints are cached, so a change there will take a while to propagate.

Reply:
https://github.com/mysteriumnetwork/payments/blob/b04a3b1325f39dacfb5d01947052a3acb97afe3c/client/client.go#L191

Reply:
This client is already used in node, so we could reuse it for this purpose as well.